dcc-submission-server/src/main/java/org/icgc/dcc/submission/service/MailService.java

```diff
@@ -92,7 +92,7 @@ public void sendSupportFeedback(@NonNull Feedback feedback) {
 
     send(
         feedback.getEmail(),
-        properties.getMail().getSupportEmail(),
+        properties.getEmail().getSupportEmail(),
```

Comment:
I know you sent a request and I unset `MAIL` on all the slaves as you asked, restarted all the slaves and it still fails.